packages/manager/modules/config/src/application/index.ts

```diff
@@ -0,0 +1,30 @@
+export type ApplicationId =
+  | 'billing'
+  | 'cloud'
+  | 'dedicated'
+  | 'exchange'
+  | 'freefax'
+  | 'hub'
+  | 'office'
+  | 'overthebox'
+  | 'pack-xdsl'
+  | 'public-cloud'
+  | 'sharepoint'
+  | 'sms'
+  | 'telecom'
+  | 'telephony'
+  | 'user'
+  | 'web';
+
+export interface Container {
+  enabled: boolean;
+  isDefault: boolean;
+  path: string;
+  containerURL: string;
+}
+
+export interface Application {
+  universe: string;
+  url: string;
+  container?: Container; // @TODO: remove nullable when 2API applications will be prodded
```

Comment:
Relates to applications#5, Isn't it? If yes, then:
```suggestion
  container: Container;
```